config/routes.rb

```diff
@@ -428,7 +428,8 @@ def patch(*)
         get "experimental" => "site_settings#index"
         get "trust-levels" => "site_settings#index"
         get "group-permissions" => "site_settings#index"
-        get "/logo-and-fonts" => "logo#index"
+        get "/logo" => "logo#index"
+        get "/fonts" => "logo#index"
```

Comment:
Should this go to `fonts#index` just for clarity? Seems there's already a controller since we have `fonts#update`.